File: Python/Start/整数的各位积和之差（Easy）.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        ans_mul = 1
        ans_plus = 0
        while(n):
            ans_mul *= n % 10
            ans_plus += n % 10
            n //= 10
        return ans_mul - ans_plus

File: C/Start/整数的各位积和之差（Easy）.c
int subtract_product_and_sum(int n) 
{
    int ans_mul = 1;
    int ans_plus = 0;
    for (; n;)
    {
        ans_mul *= n % 10;
        ans_plus += n % 10;
        n /= 10;
    }
    return ans_mul - ans_plus;
}